csr-engine/eval-kit/e2/prep_pools.py

```
@@ -0,0 +1,57 @@
+            if m.group(1) not in convs:
+                convs.append(m.group(1))
+    return convs
+
+def file_touch(target):
+    if not target:
+        return []
+    q = ("SELECT DISTINCT ce.session_id FROM code_evolution ce "
+         "WHERE ce.file_path LIKE '%' || ? "
+         "AND EXISTS (SELECT 1 FROM chunks c WHERE c.conversation_id = ce.session_id "
+         "            AND c.timestamp <= '" + FREEZE + "T23:59:59Z')")
+    out = subprocess.run([SQLITE, DB, q], input=target, capture_output=True, text=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)